trunk/SUAVE/Plots/Geometry_Plots/plot_polars.py

```diff
@@ -0,0 +1,88 @@
+## @ingroup Plots-Geometry_Plots
```

Comment:
Don't put this in geometry plots. That's not an intuitive directory. Move it a folder up.

Reply:
Addressed.